docs/thirdparty-openapi3-snippets.yaml

```diff
@@ -224,12 +257,46 @@ components:
       description: Scope + Account Identifier mapping for a Consent.
       properties:
         scope:
-          $ref: '#/components/schemas/AuthScopesType'
+          $ref: '#/components/schemas/AuthorizationScopeType'
         accountId:
           $ref: '#/components/schemas/AccountAddress'
       required:
         - scope
         - accountId
+    ThirdpartyAuthorizationIDPostResponse:
+      title: ThirdpartyAuthorizationIDPostResponse
+      type: object
+      description: 'The response for POST /thirdpartyRequests/transactions/{id}/authorizations request.'
+      properties:
+        challenge:
+          type: string
+          description: The original Challenge Object as a JSON string
+        value:
+          allOf:
+            - $ref: '#/components/schemas/ThirdpartyAuthorizationIDPutRequest/properties/value/allOf/0'
+          description: Base64 encoded binary string - the signed challenge
+        consentId:
+          allOf:
+            - $ref: '#/components/schemas/Consent/properties/id/allOf/0'
+          description: |
+            Common ID between the PISP and FSP for the Consent object This tells DFSP and auth-service which constent allows the PISP to initiate transaction.
+        sourceAccountId:
+          allOf:
+            - $ref: '#/components/schemas/AccountAddress'
+          description: 'DFSP specific account identifiers, e.g. `dfspa.alice.1234`'
+        status:
+          type: string
+          enum:
+            - VERIFIED
+          description: The status of the authorization.
+        currentState:
```

Comment:
I don't see `currentState` on any of our sequence diagrams. (ref: https://raw.githubusercontent.com/mojaloop/pisp/master/docs/out/transfer/api_calls_detailed/PISPTransferDetailedAPI-page2.png)

Reply:
https://github.com/mojaloop/thirdparty-scheme-adapter/blob/master/src/interface/api-outbound-template.yaml#L306

copied from here. not sure where/why this was introduced

Reply:
Ok maybe @sridharvoruganti or @eoln will know more about this. My recommendation is that we remove it. It could be an internal state property that the thirdparty-scheme-adapter keeps track of, and perhaps exposes to the DFSP backend, but I don't think we need it in the thirdparty api.

Reply:
@kleyow @lewisdaly currentState is only used in sync responses on Outbound

Reply:
IMHO: The Outbound schemes elements should live in separate namespaces - maybe `scheme-adapters` or `SDK`  to distinguish them from the core switch interfaces